pkg/kev/override.go

```diff
@@ -99,9 +99,9 @@ func (o *composeOverride) volumesWithLabelsMatching(other *composeOverride) Volu
 // expandLabelsFrom returns a copy of the compose override
 // filling in gaps in services and volumes labels (keys and values) using the provided override.
 func (o *composeOverride) expandLabelsFrom(other *composeOverride) *composeOverride {
-	services := o.servicesLabelsExpandedFrom(other)
+	// services := o.servicesLabelsExpandedFrom(other)
```

Comment:
Do we need this line?

Reply:
I will remove later on cleanup

Reply:
OK, let's just leave a TODO comment here to remember to revisit?